src/squares.ts

```diff
@@ -80,6 +80,33 @@ const raids: { [key: string]: string[] } = {
     "Depositing motes at the wrong relay",
     "Gambit team has no void weapons",
   ],
+  "sotp": [
```

Comment:
Change to `"Scourge of the Past"` please - I'm using full raid names as the keys, since that's what appears in the raid selection list

Reply:
Fixed